src/plugins/orangehrmRecruitmentPlugin/Dao/RecruitmentAttachmentDao.php

```diff
@@ -204,7 +231,7 @@ protected function getInterviewAttachmentPaginator(
     public function getInterviewAttachmentsCount(
         InterviewAttachmentSearchFilterParams $interviewAttachmentParamHolder
     ): int {
-        return $this->getInterviewAttachmentPaginator($interviewAttachmentParamHolder)->count();
+        return $this->getInterviewAttachmentPaginator($interviewAttachmentParamHolder)->setUseOutputWalkers(false)->count();
```

Comment:
```suggestion
        return $this->getInterviewAttachmentPaginator($interviewAttachmentParamHolder)
            ->setUseOutputWalkers(false)
            ->count();
```
preferred formatting